Moved arithematic operators to sympy/sets/sets.py

rahilhastu opened the pull request:
All the functions related to generic use for all sets are moved from
sympy/sets/setexpr.py to sympy/sets/sets.py

<!-- Your title above should be a short description of what
was changed. Do not include the issue number in the title. -->

#### References to other Issues or PRs
<!-- If this pull request fixes an issue, write "Fixes #NNNN" in that exact
format, e.g. "Fixes #1234". See
https://github.com/blog/1506-closing-issues-via-pull-requests .-->
Fixes #14366

rahilhastu committed:
Moved arithematic operators to sympy/sets/sets.py

All the functions related to generic use for all sets are moved from
sympy/sets/setexpr.py to sympy/sets/sets.py

rahilhastu committed:
Moved arithematic operators to sympy/sets/sets.py

All the functions related to generic use for all sets are moved from
sympy/sets/setexpr.py to sympy/sets/sets.py

lilithxxx commented on sympy/sets/sets.py:
```diff
@@ -1959,3 +1957,33 @@ def simplify_intersection(args):
+
+def set_function(f, x):
+    from sympy.sets.handlers.functions import _set_function
+    from sympy.sets.setexpr import _apply_operation
```
```_apply_operation``` function is not needed here so no need to import it here.

rahilhastu replied:
> _apply_operation function is not needed here so no need to import it here.

removed it. 
Is it fine now?

rahilhastu committed:
Moved arithematic operators to sympy/sets/sets.py

All the functions related to generic use for all sets are moved from
sympy/sets/setexpr.py to sympy/sets/sets.py

Upabjojr commented on sympy/sets/sets.py:
```diff
@@ -1959,3 +1957,32 @@ def simplify_intersection(args):
+
+def set_add(x, y):
+    from sympy.sets.handlers.add import _set_add
+    from sympy.sets.setexpr import _apply_operation
```
`_apply_operation` and `_handle_finite_sets` should also be moved, in my opinion.

rahilhastu replied:
do I need to move it? @lilithxxx

rahilhastu committed:
moved _apply_operation and _handle_finite_sets

Upabjojr commented:
Thanks for contributing!
